### DISCUSSION TITLE
MyRadioButton无法套用EventType和Data

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4692] [提问] [answered]
话说PCL都是一个控件一个控件绑定的吗（thinking

**Comment by MoYuan-CN** [comment]
![image](https://github.com/user-attachments/assets/7e92b897-f2ec-4753-b7ae-36fc99e4b20b)
压根没有这个属性……

**Reply by Joker2184** [reply, marked as answer]
可能会被打（
这玩意在WRF是单选按钮 实际使用并不会为其他编写者提供更大的帮助
（毕竟PCL有很多WRF控件无法单独使用（
具体参考：https://learn.microsoft.com/zh-cn/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.radiobutton?view=windows-app-sdk-1.5

**Reply by MoYuan-CN** [reply]
从命名规范上来讲确实是一个单选按钮，也确实是做了单选用途（分页）
